dev/storage_base.h

```diff
@@ -450,6 +451,18 @@ namespace sqlite_orm {
                 return this->connection->retain_count() > 0;
             }
 
+            /*
+             * checks whether there is a transaction in place
+             */
+            bool in_transaction() const {
+                auto&& conn = this->connection;
```

Comment:
what is the purpose of right reference here?

Reply:
auto&& because connection was a unique_ptr<> and I did not want to move from it

Reply:
but i removed it
now I call get_connection()